### ISSUE TITLE
java.lang.NullPointerException ARN

### ISSUE THREAD

**Issue by ChengZh98** [didomi/react-native#33]
Hi, i m having many ARN related to 

`java.lang.NullPointerException: 
  at java.util.Objects.requireNonNull (Objects.java:228)
  at com.reactnativedidomi.DidomiModule$setupUI$1.run (DidomiModule.java:278)
  at android.os.Handler.handleCallback (Handler.java:888)
  at android.os.Handler.dispatchMessage (Handler.java:100)
  at android.os.Looper.loop (Looper.java:213)
  at android.app.ActivityThread.main (ActivityThread.java:8178)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:513)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1101)`

Can you do something about it?
The logs says that the problem is located here :   
```
DidomiModule.java line 278
com.reactnativedidomi.DidomiModule$setupUI$1.run
```

Thanks.

**Comment by pmerlet-at-didomi** [comment]
Hello, we will publish a fix so it does not crash. This is probably because `SetupUI` is called at a time when your app is not active (`currentActivity` returns `null`), so you may check if notice is correctly displayed after the fix. It is possible that the call to `SetupUI` is done while the application is not ready yet, or when the user already left the app.

**Comment by pmerlet-at-didomi** [comment]
Should be fixed in version 1.7.0
